[Client] Error: Cannot find module 'node-cache'

If node-cache is not installed globally the client throws this error `Error: Cannot find module 'node-cache'`, because node-cache is not listed in dependencies.

--- a/promptic-client/package.json
+++ b/promptic-client/package.json
@@ -21,7 +21,8 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "axios": "^1.6.2"
+    "axios": "^1.6.2",
+    "node-cache": "^5.1.2"
   },
   "devDependencies": {
     "@types/jest": "^29.5.10",

--- a/promptic-client/src/index.ts
+++ b/promptic-client/src/index.ts
@@ -2,7 +2,7 @@ import axios, { AxiosInstance } from "axios";
 import NodeCache from "node-cache";
 
 export interface PrompticClientConfig {
-  baseUrl: string;
+  baseUrl?: string;
   projectKey: string;
 }
 
@@ -23,7 +23,7 @@ export class PrompticClient {
   private cacheRefreshIntervalMs: number;
 
   constructor(config: PrompticClientConfig) {
-    this.baseUrl = config.baseUrl;
+    this.baseUrl = config.baseUrl || "https://api.promptic.app";
     this.projectKey = config.projectKey;
     this.cacheRefreshIntervalMs = 30000;
     this.client = axios.create({

--- a/web/app/docs/code-examples/page.tsx
+++ b/web/app/docs/code-examples/page.tsx
@@ -39,7 +39,6 @@ import { PrompticClient } from "promptic-client";
 
 // Initialize the client with your Promptic base URL
 const client = new PrompticClient({
-  baseUrl: "https://api.promptic.app",
   projectKey: "59e5d2ec-405c-4868-a4f1-701d90fd1e94",
 });
 

--- a/web/app/docs/page.tsx
+++ b/web/app/docs/page.tsx
@@ -39,7 +39,6 @@ import { PrompticClient } from "promptic-client";
 
 // Initialize the client with your Promptic base URL
 const client = new PrompticClient({
-  baseUrl: "https://api.promptic.app",
   projectKey: "59e5d2ec-405c-4868-a4f1-701d90fd1e94",
 });
 

--- a/web/constants/codeSnippets.ts
+++ b/web/constants/codeSnippets.ts
@@ -5,7 +5,6 @@ export const CLIENT_USAGE_CODE = `import { PrompticClient } from "promptic-clien
 
 // Initialize the client with your Promptic base URL
 const client = new PrompticClient({
-  baseUrl: "https://api.promptic.app",
   projectKey: "YOUR_PROJECT_KEY", // Get this from the Settings tab
 });
 
@@ -17,7 +16,6 @@ export const generateClientCodeWithKey = (projectKey: string) => {
 
 // Initialize the client with your Promptic base URL
 const client = new PrompticClient({
-  baseUrl: "https://api.promptic.app",
   projectKey: "${projectKey}",
 });
 

